site/content/blog/2020-03-10-emojis.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "All About Emojis"
+slug: all-about-emojis
+date: 2020-03-10T12:00:00-05:00
+categories:
+    - "markdown"
+author: Harrison Healey
+github: hmhealey
+community: harrison
+---
+
+Emojis are a great way to add tone to a piece of text. In person, you can easily tell someone's mood based on their body language and how they speak, but that's much more difficult with text alone. They also help make text-based conversation feel more casual, relaxed, and fun. Thanks to emojis, we can chat with much more real emotion than you might get by being careful about your word choice or by including just the right number of exclamation marks and periods at the end of a sentence.
+
+### So where did emojis come from?
+
+Emojis (or "emoji" which is technically the correct pluralization) are characters that are displayed as small images on devices that support them. This makes them different from emoticons which are loosely defined, purely text-based sequences of characters, such as <3 or ಠ_ಠ. The term emoji comes from the Japanese words e- (meaning "pictures") and -moji (meaning "character") since the original emojis were created by Japanese cell phone carriers in the 1990s. Eventually, the Unicode Consortium stepped in and came up with the standard set of emojis that we use today, although that set is frequently updated and expanded from the original version.
```

Comment:
```suggestion
Emojis (or "emoji" which is technically the correct pluralization) are characters that are displayed as small images on devices that support them. This makes them different from emoticons which are loosely defined, purely text-based sequences of characters, such as ``<3`` or ``ಠ_ಠ``. The term emoji comes from the Japanese words e- (meaning "pictures") and -moji (meaning "character") since the original emojis were created by Japanese cell phone carriers in the 1990s. Eventually, the Unicode Consortium stepped in and came up with the standard set of emojis that we use today, although that set is frequently updated and expanded from the original version.
```